blocktx/store/sql/insert_block_transactions.go

```diff
@@ -7,25 +7,14 @@ import (
 	"strings"
 
 	"github.com/bitcoin-sv/arc/blocktx/blocktx_api"
-	"github.com/ordishs/gocore"
 )
 
 // InsertBlockTransactions inserts the transaction hashes for a given block hash
 func (s *SQL) InsertBlockTransactions(ctx context.Context, blockId uint64, transactions []*blocktx_api.TransactionAndSource, merklePaths []string) error {
-	start := gocore.CurrentNanos()
-	defer func() {
-		gocore.NewStat("blocktx").NewStat("InsertBlockTransactions").AddTime(start)
-	}()
-
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	qTx, err := s.db.Prepare(`
-			INSERT INTO transactions (hash, merkle_path) VALUES ($1, $2)
-			ON CONFLICT(hash) DO UPDATE SET merkle_path=$2
-			RETURNING id
-			;
-		`)
+	qTx, err := s.db.Prepare(`INSERT INTO transactions (hash, merkle_path) VALUES ($1, $2);`)
```

Comment:
With this change the transaction can not be "upserted" any more. That may be the reason why the e2e test fails.

Reply:
e2e tests don't use embedded db. They have completely different flow.